runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -515,10 +519,15 @@ SOLINK_FLAGS+=$(SOLINK_FLAGS_EXTRA)
 #
 PROTO_CMD?=protoc
 
-SOLINK_SLINK+=protobuf ssl
-
 CXX_DEFINES+=GOOGLE_PROTOBUF_NO_RTTI
 
+ifeq ($(JITAAS_ENABLE_SSL),)
+SOLINK_SLINK+=protobuf
```

Comment:
Since this change is temporary, I tried to keep the diff to the minimum. But on a second thought, it's good to keep the style consistent. Will update the code next.